src/ui/proposal/ProposalCardContent.tsx

```diff
@@ -96,8 +97,8 @@ function ProposalCardContent({
 
 	// Replace the whole card if the reviewAnnotation.error is acknowledgeable.
 	if (
-		reviewAnnotation.error &&
-		reviewAnnotation.error.recovery === RecoveryOption.ACKNOWLEDGEABLE
+		typeof reviewAnnotation.error !== 'number' &&
+		reviewAnnotation?.error.recovery === RecoveryOption.ACKNOWLEDGEABLE
```

Comment:
I think this is causing hardcrash in proposals.

Reply:
Yes, you are right Alberto, thanks, it should be changed to `reviewAnnotation.error?.recovery`